FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -230,31 +247,53 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <note>Contrary to <bibl><ptr target="bm:Marrassini1986Laurenziana"/><citedRange unit="page">93</citedRange></bibl>, who considered textual unit 1 (Miracles of Mary) on 
                                     <locus from="2" to="113"/> of different hand as textual unit 2 (Miracles of ʾEwosṭātewos) on <locus from="113vb" to="122v"/> and other notes on <locus from="123ra" to="128v"/></note>
                             </handNote>     </handDesc>
-                            <objectDesc></objectDesc>
-                            <addition></addition>
-                            <history></history>
-                        <msPart xml:id="p2"><msIdentifier><msContents><summary><msItem xml:id="p2_i1">
+                            </physDesc>
+                            
+                            </msPart>
+                        <msPart xml:id="p2"><msIdentifier/><msContents><summary/><msItem xml:id="p2_i1">
                         <locus from="3ra" to="6va"></locus>
-                            <layout columns="2" writtenLines="25"></layout>
-                            <title type="incomplete" ref="LIT2384Taamme">Miracles of Mary<title/>
+                            
+                            <title type="incomplete" ref="LIT2384Taamme">Miracles of Mary</title>
                                 <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ።
-                                    እምቅድመ፡ አምር፡ ይትነበብ፡ መቅድመ፡ ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ዘበዕብራይስጢ፡ ማሪሃም፡ ዝንቱ፡ ሥርዓት፡ ዘወጽአ፡ እመካነ፡ ማልቃ፡ ዘምስር።</incipitxm>
+                                    እምቅድመ፡ አምር፡ ይትነበብ፡ መቅድመ፡ ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ዘበዕብራይስጢ፡ ማሪሃም፡ ዝንቱ፡ ሥርዓት፡ ዘወጽአ፡ እመካነ፡ ማልቃ፡ ዘምስር።</incipit>
                             <note>Two introductions to the text similar to <bibl><ptr target="bm:Budge1900Miracles"/><citedRange unit="page">2</citedRange></bibl>, 
                                 <bibl><ptr target="bm:Budge1900Miracles"/><citedRange unit="page">4-9</citedRange></bibl> 
                                 and <bibl><ptr target="bm:Budge1900Miracles"/><citedRange unit="page">14</citedRange></bibl>.</note>
+                        </msItem>
+                        </msContents>
+                            <physDesc>
+                                <objectDesc>
+                                    <layoutDesc><layout columns="2" writtenLines="25"></layout></layoutDesc>
+                                </objectDesc>
                             <handDesc><handNote xml:id="h2" script="Ethiopic">
                                 <ab type="script"/>
                                 <locus from="3ra" to="6va"/>
-                            </handNote>    </handDesc>
-                    <msPart xml:id="p3"><msIdentifier><msContents><summary><msItem xml:id="p3_i1">
+                            </handNote>    </handDesc></physDesc></msPart>
+                    <msPart xml:id="p3"><msIdentifier/><msContents><summary/><msItem xml:id="p3_i1">
```

Comment:
msIdentifier must be a closed empty tag (either `<msIdentifier></msIdentifier>` or simply `<msIdentifier/>`)
summary must be a closed empty tag (either `<summary></summary>` or simply `<summary/>`)